src/integration-tests/introspector/checkBeans.inputt

```diff
@@ -22,6 +22,8 @@
 
 /ServerTemplates/mycluster-template/Log/mycluster-template,FileName,logs/mycluster-template.log,/shared/logs/managed-server${id}.log
 /ServerTemplates/mycluster-template,ListenAddress,None,domain1-managed-server${id}
+# Note that a customer/admin should never override a port, they must stay fixed
+/ServerTemplates/mycluster-template,ListenPort,8001,8001
```

Comment:
Notes (same as in slack):

- Ports are not overridden by the introspector, only read.
- introspectTest.sh does not start the operator (it emulates it), so this test is insufficient for testing whether the operator is setting up services correctly on wl pods.   See  the README in same directory for test info.   See integration-tests/README.md for info on tests that do start the 'real' operator.
- If you want to verify overrides are working for naps using this test, there (a) needs to be naps in the config, and (b) configure a 'bad' listen address in the nap, (c) verify that the before value is the 'bad' value and the after value is as expected (domain1-managed-server${id}) -- see the ListenAddress line#24 for an example.    This will only verify that the overrides are working correctly -- it won't verify that the related service was setup correctly by the operator...